deployment/5_deploy_modules.js

```diff
@@ -34,113 +34,134 @@ const deploy = async (network) => {
   console.log(config);
 
   // //////////////////////////////////
-  // Deploy Storage
```

Comment:
Since this is no longer an upgrade script, the modules/features being deployed here should no longer be referred as "new"